modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -65,6 +67,58 @@ interface ISaveFDSFieldsModalContentProps {
 	saveFDSFieldsURL: string;
 }
 
+interface IContentRendererProps {
+	cetRenderers?: FDSClientExtensionCellRenderer[];
+	item: IFDSField;
+	query: string;
+}
+
+const getRendererLabel = ({
+	cetRenderers = [],
+	rendererName,
+}: {
+	cetRenderers?: FDSClientExtensionCellRenderer[];
+	rendererName: string;
+}): string => {
+	const AVAILABLE_CELL_RENDERERS = [
+		...FDS_INTERNAL_CELL_RENDERERS,
+		...cetRenderers,
+	];
```

Comment:
It should make the code more readable, and have slightly better performance, if we filter it in separate steps:
1. First filter only internal renderers, checking only `name`. If there is a match, return `renderer.label`.
2. If no match, filter CET renderers, checking only `erc`. If there is a match, return.. I'm not sure, `cet.name`?
3. If no match in two filters above, return the stored string as it is.